tests/test_dimension.py

```diff
@@ -1958,18 +1958,18 @@ def test_topofusion_w_subdims_conddims(self):
 
         # Check generated code -- expect the gsave equation to be scheduled together
         # in the same loop nest with the fsave equation
-        bns, _ = assert_blocking(op, {'x0_blk0', 'x1_blk0', 'ix0_blk0'})
+        bns, _ = assert_blocking(op, {'x0_blk0', 'x2_blk0', 'x1_blk0'})
```

Comment:
wow, was not expecting this. Is it something we want?

Reply:
Tbh they could be in any order since it's a set. This improves code readability when you have subdimensions with long names, since the `i` prefix is not actually needed

Reply:
yes